src/azure-cli-core/azure/cli/core/commands/_introspection.py

```diff
@@ -8,50 +8,54 @@
 
 
 def extract_full_summary_from_signature(operation):
-    """ Extract the summary from the doccomments of the command. """
+    """ Extract the summary from the docstring of the command. """
     lines = inspect.getdoc(operation)
     regex = r'\s*(:param)\s+(.+?)\s*:(.*)'
     summary = ''
     if lines:
         match = re.search(regex, lines)
-        if match:
-            summary = lines[:match.regs[0][0]]
-        else:
-            summary = lines
+        summary = lines[:match.regs[0][0]] if match else lines
```

Comment:
Looks good, though I don't see any reason for why the second should be preferred.  It is just nested blocks on a single line...